File: 2710-remove-trailing-zeros-from-a-string/2710-remove-trailing-zeros-from-a-string.cs
public class Solution {
    public string RemoveTrailingZeros(string num) {
        if (num[num.Length-1] != '0') {
            return num;
        }
        
        var index = num.Length - 1;
      
        for (int i = num.Length - 2; i > -1; i--) {
            
            if (num[i] == '0') {
                index -= 1;
            } else {
                break;
            }
        }
        
        return num.Substring(0, index);
            
        
    }
}

File: 2710-remove-trailing-zeros-from-a-string/2710-remove-trailing-zeros-from-a-string.py
class Solution:
    def removeTrailingZeros(self, num: str) -> str:
        if num[-1] != "0":
            return num
        index = len(num) - 1
      
        for i in range(len(num)-2, -1, -1):
            if num[i] == "0":
                index -= 1
            else:
                break
      
        
        return num[:index]
 
        